src/components/render-page/word-counter.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import wordsCount from 'words-count'
+
+const EXCLUDED_TAGS = ['img', 'pre', 'nav']
+
+export const countWordsRecursive = (node: Element | ChildNode): number => {
+  let words = 0
+  if (
+    !node ||
+    EXCLUDED_TAGS.includes(node.nodeName.toLowerCase()) ||
+    (node as HTMLElement).classList?.contains('katex-mathml')
```

Comment:
Can you extract both of these conditions into methods that say what you want to achieve with this conditons? Like `is ignored html tag`

Reply:
changed